### ISSUE TITLE
Brave using higher cpu after the update (Linux)!!!

### ISSUE THREAD

**Issue by BIGBEASTISHANK** [brave/brave-browser#32691]
I am on arch linux 6.4.12-arch1-1
Couple weeks ago I got an update for brave and since then whenever I use brave to watch youtube  the cpu goes kaboom, i tried cleaning my physical pc, removed all dust (their were none) and cleaned everything then i freshly installed arch again and then also it is fuckingup

**Comment by Saikur0n** [comment]
I confirm this behavior also on my machine - Asus Vivobook Pro 14 Windows 11 x64 22H2 with latest august patch. I wanted to switch to brave, but after installation of 1.57.57 x64 and first launch, the fans of my laptop started to work hard in order to cool CPU, while this didn't happen with chrome, edge or Firefox. Then I tried to play YouTube video and it became even worse. Seemed to me that I launched some heavy game. 😂  Unsatisfied and uninstalled unfortunately.
Specs of my machine:
i5 11300H / 16GB Ram 3200 / Intil Iris Xe and GTX 1650 Max-Q. 😅
